### ISSUE TITLE
fails to run without clustering

### ISSUE THREAD

**Issue by sophiegvrm** [nmclark2/SCION#2]
Hello,

I am about to use SCION v3.3 without the clustering step to build a GRN. Running the tool I have faced with the following errors:
```
Warning: Error in rownames: object 'myclustdata' not found
  48: rownames [SCION.R#36]
  46: SCION [SCION.R#36]
  45: observe [C:/Users/full_path/SCION-3.3/app.R#79]
  44: <observer>
   1: runApp
```
For v3.3-hot-fix:
```
Warning: Error in rownames: object 'myclustdata' not found
  52: rownames [SCION.R#39]
  50: SCION [SCION.R#39]
  49: observe [C:/Users/full_path/SCION-3.3.h/app.R#79]
  48: <observer>
   5: runApp
   4: eval [C:/Users/full_path/SCION-3.3.h/START_SCION.R#27]
   3: eval
   1: source
```
### Steps to reproduce the issue
1. Load all input files
2. Choose Clustering -> None
4. Set up Clustering Threshold = 0
5. Leave Clustering Matrix and Clusters Files windows empty
6. Run
### Attempts to solve
I haven`t try to debug the original code by myself. Instead, I tested these settings on other versions and observed these issues only in v3.3 and v3.3-hot-fix. All works fine with previous releases.
### System details
RStudio: 2022.07.2+576 "Spotted Wakerobin"
R version: 4.2.2
OS version: Windows 10 Enterprise, version 21H2, build 19044.2251

**Comment by nmclark2** [comment]
Hi! Thanks very much for catching this! It seems when I fixed the row name issue in version 3.3 and beyond, I did not include a flag for when there is no clustering (oops!) This error is occurring because SCION is trying to check the row names in the clustering matrix which, in your case, does not exist. This should be a pretty quick fix. I will test on my end and re-push the version 3.3 hotfix. When I do so, I will let you know here and close the issue so you know that it is resolved. I don't imagine this taking longer than a day to fix. Thanks again!
